Gopkg.lock

```diff
@@ -696,7 +696,7 @@
 
 [[projects]]
   branch = "master"
-  digest = "1:11d290de3457882172ce8d9ffe930999cfb62929b58e486e1ff1b6adcf3c52bc"
+  digest = "1:d3cc042f4e05e3edff799edb30d84c510d0c5f433a1ba5a139e0a88ec5f6b5a9"
```

Comment:
Since this already includes a Gopkg.lock update, do you mind pinning x/text to a specific version in Gopkg.toml so that we don't have other future breakage?

Reply:
Done